uweb3/libs/sqltalk/safe_cookie/connection.py

```diff
@@ -6,6 +6,11 @@ def __init__(self, request, cookies, cookie_salt, autocommit=True, **kwargs):
         self.cookie_salt = cookie_salt
         self.uncommitted_cookies = []
 
+        if kwargs.pop("debug", False):
```

Comment:
`self.debug = kwargs.pop("debug", False)` should be enough

Reply:
Now defaults to True.